Xamarin.Forms.Core/SwipeView.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.ComponentModel;
+using Xamarin.Forms.Platform;
+
+namespace Xamarin.Forms
+{
+	[ContentProperty("Content")]
+	[RenderWith(typeof(_SwipeViewRenderer))]
+	public class SwipeView : ContentView, IElementConfiguration<SwipeView>
+	{
+		readonly Lazy<PlatformConfigurationRegistry<SwipeView>> _platformConfigurationRegistry;
+
+		public SwipeView()
+		{
+			_platformConfigurationRegistry = new Lazy<PlatformConfigurationRegistry<SwipeView>>(() => new PlatformConfigurationRegistry<SwipeView>(this));
+		}
+
+		public static readonly BindableProperty LeftItemsProperty = BindableProperty.Create(nameof(LeftItems), typeof(SwipeItems), typeof(SwipeView), null, BindingMode.OneWay, null);
+		public static readonly BindableProperty RightItemsProperty = BindableProperty.Create(nameof(RightItems), typeof(SwipeItems), typeof(SwipeView), null, BindingMode.OneWay, null);
+		public static readonly BindableProperty TopItemsProperty = BindableProperty.Create(nameof(TopItems), typeof(SwipeItems), typeof(SwipeView), null, BindingMode.OneWay, null);
+		public static readonly BindableProperty BottomItemsProperty = BindableProperty.Create(nameof(BottomItems), typeof(SwipeItems), typeof(SwipeView), null, BindingMode.OneWay, null);
```

Comment:
still no implicit collection support. add a defaultvaluecreator

Reply:
Added defaultvaluecreator and an extra sample using SwipeItems from resource: https://github.com/xamarin/Xamarin.Forms/blob/swipeview-II/Xamarin.Forms.Controls/GalleryPages/SwipeView/ResourceSwipeItemsGallery.xaml#L49